internal/auth/rhsso_authenticator.go

```diff
@@ -55,7 +55,7 @@ func (rh *RHSSOAuthenticator) parseToken(userToken *jwt.Token) (User, error) {
 	}
 
 	return User{
-		Username:     claims["username"].(string),
+		Username:     claims["preffered_username"].(string),
 		Organization: claims["org_id"].(string),
 		ClientID:     claims["client_id"].(string),
```

Comment:
I don't se client ID as part of the token, why do you think we need it?

Reply:
It was in the doc but I don't think we need it. either username or `sub`